pom.xml

```diff
@@ -77,17 +93,6 @@
       <version>RELEASE</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
Is this not needed anymore, or was deleted accidentally?

Reply:
I think it is not deleted actually, there is an other one in the file.

Reply:
You're right, they are duplicated. Seems like a mistake from some previous merge.